pysgrid/utils.py

```diff
@@ -187,16 +187,16 @@ def calculate_bearing(lon_lat_1, lon_lat_2):
     return bearing from true north in degrees
 
     """
-    lon_lat_1_radians = lon_lat_1 * np.pi/180
-    lon_lat_2_radians = lon_lat_2 * np.pi/180
+    lon_lat_1_radians = lon_lat_1 * np.pi / 180
+    lon_lat_2_radians = lon_lat_2 * np.pi / 180
     lon_1 = lon_lat_1_radians[..., 0]
     lat_1 = lon_lat_1_radians[..., 1]
     lon_2 = lon_lat_2_radians[..., 0]
     lat_2 = lon_lat_2_radians[..., 1]
-    x1 = np.sin(lon_2-lon_1) * np.cos(lat_2)
-    x2 = np.cos(lat_1)*np.sin(lat_2) - np.sin(lat_1)*np.cos(lat_2)*np.cos(lon_2-lon_1)  # noqa
+    x1 = np.sin(lon_2 - lon_1) * np.cos(lat_2)
+    x2 = np.cos(lat_1) * np.sin(lat_2) - np.sin(lat_1) * np.cos(lat_2) * np.cos(lon_2 - lon_1)  # noqa
     bearing_radians = np.arctan2(x1, x2)
-    bearing_degrees = bearing_radians * 180/np.pi
+    bearing_degrees = bearing_radians * 180 / np.pi
```

Comment:
Not a problem, but these changes only causes noise for the reviewer and they were not necessary.